packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/gollm/GoLLMController.java

```diff
@@ -83,7 +83,7 @@ void init() {
 	})
 	public ResponseEntity<TaskResponse> createModelCardTask(
 			@RequestParam(name = "document-id", required = true) final UUID documentId,
-			@RequestParam(name = "mode", required = false, defaultValue = "async") final TaskMode mode) {
+			@RequestParam(name = "mode", required = false, defaultValue = "ASYNC") final TaskMode mode) {
```

Comment:
These _should_ be fixed to work with lowercase on the latest version of `main`. Sorry about that. apparently defining an enum as `TYPE("type")` in java does nothing.

Reply:
I just tried on the latest version of main with the lower cases and am still getting issues with the conversion:
`Failed to convert value of type 'java.lang.String' to required type 'software.uncharted.terarium.hmiserver.service.tasks.TaskService$TaskMode'; Failed to convert from type [java.lang.String] to type [@org.springframework.web.bind.annotation.RequestParam software.uncharted.terarium.hmiserver.service.tasks.TaskService$TaskMode] for value [async]`

Reply:
Argh I guess my change didn't make it into `main`. Pushed it to this branch. Can you give it another try @blanchco

Reply:
Hold on that's another enum. lol. Java enums are ridiculous

Reply:
Okay Spring is stupid and doesn't deal with enums in any sane way. Leave these as upper case.